### PR TITLE
Made all programs and summary served directly from json file

### PR DESCRIPTION
**Pull request by ywkim312**
Currently, the allprograms and summary data are provided from the database. Creating and updating the database of these has been done by csv file and not from json. Now, the new updates of these information is done directly as json format so it is harder to update the database, and since it is already json format, it will be much beneficial to use json directly instead of querying from the database in terms of the updating process and serving via end point.

### CODE REVIEW

File: app/controllers/pdl.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 from app.models.allprograms import AllProgram
 
 
+LANDING_PAGE_DATA_PATH = os.path.join("controllers", "data", "landingpage")
+ALLPROGRAM_DATA_JSON = "allPrograms.json"
```
Comment (sandeep-ps) [inline app/controllers/pdl.py:15]: This must be changed to "allprograms.json" to match the current file name. 
On a related note, separating words in filenames by underscores can give better readability.

Comment (ywkim312) [reply app/controllers/pdl.py]: updated. I agree. All the file names used in here are the same ones as the filenames in box folder. I also prefer using underscore.

File: app/controllers/pdl.py [hunk 2]
```diff
@@ -11,6 +11,9 @@
 from app.models.allprograms import AllProgram
 
 
+LANDING_PAGE_DATA_PATH = os.path.join("controllers", "data", "landingpage")
+ALLPROGRAM_DATA_JSON = "allprograms.json"
+SUMMARY_DATA_JSON = "summary.json"
 CSP_JSON_DATA_PATH = os.path.join("controllers", "data", "conservation", "csp")
 CSP_MAP_DATA_JSON = "csp_map_data.json"
 CSP_STATE_DISTRIBUTION_DATA_JSON = "csp_state_distribution_data.json"
```
Comment (sandeep-ps) [inline app/controllers/pdl.py:19]: Please remove EQIP_JSON_DATA_PATH in line 24. It's a repeated entry, and the path in that line is incorrect. It looks like this change crept in a few weeks ago. I might have missed testing other endpoints as part of that PR review.

Comment (ywkim312) [reply app/controllers/pdl.py:19]: removed. thanks for the good catch

Comment (sandeep-ps) [reply app/controllers/pdl.py:19]: Thank you.

File: app/controllers/pdl.py (hunk 1 above)
Comment (sandeep-ps) [reply app/controllers/pdl.py]: Thanks.

File: CHANGELOG.md [hunk 3]
```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Update SNAP JSON file structure and add participation percentages. [#48](https://github.com/policy-design-lab/pdl-api/issues/48)
+- Make summary and all programs directly served from json file. [#65](https://github.com/policy-design-lab/pdl-api/issues/65)
```
Comment (sandeep-ps) [inline CHANGELOG.md:32]: Please move this to the unreleased section. The rest of the changes look good. Thanks.

Comment (ywkim312) [reply CHANGELOG.md]: updated

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**